### PR TITLE
docs: Update README to display prominent features and link to a new dedicated features page

### PR DESCRIPTION
**Pull request by silentoplayz**
# Pull Request Checklist

### Note to first-time contributors: Please open a discussion post in [Discussions](https://github.com/open-webui/open-webui/discussions) and describe your changes before submitting a pull request.

**Before submitting, make sure you've checked the following:**

- [x] **Target branch:** Please verify that the pull request targets the `dev` branch.
- [x] **Description:** Provide a concise description of the changes made in this pull request.
- [x] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [x] **Documentation:** Have you updated relevant documentation [Open WebUI Docs](https://github.com/open-webui/docs), or other documentation sources?
- [x] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?
- [x] **Testing:** Have you written and run sufficient tests for validating the changes?
- [x] **Code review:** Have you performed a self-review of your code, addressing any coding standard issues and ensuring adherence to the project's coding standards?
- [x] **Label:** To cleary categorize this pull request, assign a relevant label to the pull request title, using one of the following:
  - **BREAKING CHANGE**: Significant changes that may affect compatibility
  - **build**: Changes that affect the build system or external dependencies
  - **ci**: Changes to our continuous integration processes or workflows
  - **chore**: Refactor, cleanup, or other non-functional code changes
  - **docs**: Documentation update or addition
  - **feat**: Introduces a new feature or enhancement to the codebase
  - **fix**: Bug fix or error correction
  - **i18n**: Internationalization or localization changes
  - **perf**: Performance improvement
  - **refactor**: Code restructuring for better maintainability, readability, or scalability
  - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)
  - **test**: Adding missing tests or correcting existing tests
  - **WIP**: Work in progress, a temporary label for incomplete or ongoing work

# Changelog Entry

### Description

- Revamped features list on `README.md` page into a smaller list of the key features of Open Webui, with the remainder of remarked features moved over to https://docs.openwebui.com/features/

### Added

- docs/features.md page added on the docs repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,97 +11,39 @@
 [![Discord](https://img.shields.io/badge/Discord-Open_WebUI-blue?logo=discord&logoColor=white)](https://discord.gg/5rJgQTnV4s)
 [![](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=%23fe8e86)](https://github.com/sponsors/tjbck)
 
-Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI designed to operate entirely offline. It supports various LLM runners, including Ollama and OpenAI-compatible APIs. For more information, be sure to check out our [Open WebUI Documentation](https://docs.openwebui.com/).
```
Comment (tjbck) [inline README.md:14]: Let's keep this part.

File: README.md [hunk 2]
```diff
@@ -11,97 +11,39 @@
...
 
 ![Open WebUI Demo](./demo.gif)
 
-## Features ⭐
+## Key Features of Open WebUI ⭐
 
-- 🖥️ **Intuitive Interface**: Our chat interface takes inspiration from ChatGPT, ensuring a user-friendly experience.
+- 📚 **Local RAG Integration**: Dive into the future of chat interactions with the groundbreaking Retrieval Augmented Generation (RAG) support. This feature seamlessly integrates document interactions into your chat experience. You can load documents directly into the chat or add files to your document library, effortlessly accessing them using `#` command in the prompt. In its alpha phase, occasional issues may arise as we actively refine and enhance this feature to ensure optimal performance and reliability. **Revolutionize chat interactions with RAG support**.
```
Comment (tjbck) [inline README.md:20]: The descriptions should be shorter and more concise.

I've curated the key features:

1. Effortless Setup
2. Ollama/OpenAI API Integration
3. Responsive Design
4. Full Markdown/LaTeX and Code Syntax Highlighting Support
5. Model Builder
6. Local RAG Integration
7. Web Browsing Capability
8. Image Generation Integration
9. Many Model Conversations
10. Role-Based Access Control (RBAC)
11. Multilingual Support
12. Continuous Updates

Comment (silentoplayz) [reply README.md:20]: Will work on it.

File: README.md [hunk 3]
```diff
@@ -15,93 +15,33 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
-- 📧 **Trusted Email Authentication**: Authenticate using a trusted email header, adding an additional layer of security and authentication.
-
 - 🔐 **Role-Based Access Control (RBAC)**: Ensure secure access with restricted permissions; only authorized individuals can access your Ollama, and exclusive model creation/pulling rights are reserved for administrators.
 
-- 🔒 **Backend Reverse Proxy Support**: Bolster security through direct communication between Open WebUI backend and Ollama. This key feature eliminates the need to expose Ollama over LAN. Requests made to the '/ollama/api' route from the web UI are seamlessly redirected to Ollama from the backend, enhancing overall system security.
-
 - 🌐🌍 **Multilingual Support**: Experience Open WebUI in your preferred language with our internationalization (i18n) support. Join us in expanding our supported languages! We're actively seeking contributors!
 
-- 🌟 **Continuous Updates**: We are committed to improving Open WebUI with regular updates and new features.
+- 🌟 **Continuous Updates**: We are committed to improving Open WebUI with regular updates, fixes, and new features. **Enjoy the latest innovations in chat technology**.
```
Comment (tjbck) [inline README.md:42]: I'd drop `**Enjoy the latest innovations in chat technology**` 😂

File: README.md [hunk 4]
```diff
@@ -15,93 +15,33 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
-
-- 📱 **Responsive Design**: Enjoy a seamless experience on both desktop and mobile devices.
-
-- ⚡ **Swift Responsiveness**: Enjoy fast and responsive performance.
+## Key Features of Open WebUI ⭐
 
 - 🚀 **Effortless Setup**: Install seamlessly using Docker or Kubernetes (kubectl, kustomize or helm) for a hassle-free experience.
 
-- 🌈 **Theme Customization**: Choose from a variety of themes to personalize your Open WebUI experience.
+- 🤝 **OpenAI API Integration**: Effortlessly integrate OpenAI-compatible APIs for versatile conversations alongside Ollama models. Customize the Ollama API URL to link with **LMStudio, GroqCloud, Mistral, OpenRouter, and more**.
```
Comment (tjbck) [inline README.md:22]: Ollama/OpenAI API Integration

**Comment by silentoplayz** [comment]
Anymore changes between this and https://github.com/open-webui/docs/pull/82 that you would like me to make?

File: README.md [hunk 5]
```diff
@@ -15,93 +15,33 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
 ![Open WebUI Demo](./demo.gif)
 
-## Features ⭐
+## Key Features of Open WebUI ⭐
 
-- 🖥️ **Intuitive Interface**: Our chat interface takes inspiration from ChatGPT, ensuring a user-friendly experience.
+- 🚀 **Effortless Setup**: Install seamlessly using Docker or Kubernetes (kubectl, kustomize or helm) for a hassle-free experience with support for both ':ollama' and ':cuda' tagged images.
 
-- 📱 **Responsive Design**: Enjoy a seamless experience on both desktop and mobile devices.
+- 🤝 **Ollama/OpenAI API Integration**: Effortlessly integrate OpenAI-compatible APIs for versatile conversations alongside Ollama models. Customize the Ollama API URL to link with **LMStudio, GroqCloud, Mistral, OpenRouter, and more**.
```
Comment (tjbck) [inline README.md:22]: > Customize the Ollama API URL to link with **LMStudio, GroqCloud, Mistral, OpenRouter, and more**

should be `OpenAI API`

File: README.md [hunk 6]
```diff
@@ -15,93 +15,33 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
 
-- ⚡ **Swift Responsiveness**: Enjoy fast and responsive performance.
-
-- 🚀 **Effortless Setup**: Install seamlessly using Docker or Kubernetes (kubectl, kustomize or helm) for a hassle-free experience.
-
-- 🌈 **Theme Customization**: Choose from a variety of themes to personalize your Open WebUI experience.
-
-- 💻 **Code Syntax Highlighting**: Enjoy enhanced code readability with our syntax highlighting feature.
+- 📱 **Responsive Design**: Enjoy a seamless experience across Desktop PC, Laptop, and Mobile devices.
```
Comment (tjbck) [inline README.md:24]: We can mention PWA support here

File: README.md [hunk 7]
```diff
@@ -15,93 +15,33 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
 - 🔐 **Role-Based Access Control (RBAC)**: Ensure secure access with restricted permissions; only authorized individuals can access your Ollama, and exclusive model creation/pulling rights are reserved for administrators.
 
-- 🔒 **Backend Reverse Proxy Support**: Bolster security through direct communication between Open WebUI backend and Ollama. This key feature eliminates the need to expose Ollama over LAN. Requests made to the '/ollama/api' route from the web UI are seamlessly redirected to Ollama from the backend, enhancing overall system security.
-
 - 🌐🌍 **Multilingual Support**: Experience Open WebUI in your preferred language with our internationalization (i18n) support. Join us in expanding our supported languages! We're actively seeking contributors!
 
-- 🌟 **Continuous Updates**: We are committed to improving Open WebUI with regular updates and new features.
+- 🌟 **Continuous Updates**: We are committed to improving Open WebUI with regular updates, fixes, and new features.
+
+Want to learn more about Open WebUI's features? Check out our [Open WebUI documentation](https://docs.openwebui.com/) for a comprehensive overview!
```
Comment (tjbck) [inline README.md:44]: https://docs.openwebui.com/features

File: README.md [hunk 8]
```diff
@@ -15,93 +15,35 @@ Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI d
...
 
-- 🔄 **Update All Ollama Models**: Easily update locally installed models all at once with a convenient button, streamlining model management.
+- 📚 **Local RAG Integration**: Dive into the future of chat interactions with groundbreaking Retrieval Augmented Generation (RAG) support. This feature seamlessly integrates document interactions into your chat experience. You can load documents directly into the chat or add files to your document library, effortlessly accessing them using the `#` command before a query.
 
-- ⬆️ **GGUF File Model Creation**: Effortlessly create Ollama models by uploading GGUF files directly from the web UI. Streamlined process with options to upload from your machine or download GGUF files from Hugging Face.
+- 🌐 **Web Browsing Capability**: Seamlessly integrate websites into your chat experience using the `#` command followed by a URL. This feature allows you to incorporate web content directly into your conversations, enhancing the richness and depth of your interactions.
```
Comment (tjbck) [inline README.md:34]: Now that we have  web search feature as well, maybe say `Web Search/Browsing`?

Comment (silentoplayz) [reply README.md:36]: Added

**Comment by tjbck** [comment]
LGTM, Thanks!

**Pull request merged**